cmd/libvirt-actuator/README.md

```diff
@@ -2,28 +2,114 @@
+   - update qemu-kvm to at least 2.4
+   - create default volume storage
+   - create cloud init volume
+   - create `fedora_base` volume
+
+   ```sh
+   $ bash cmd/libvirt-actuator/resources/init.sh
+   ```
+
+   In case the default [cloud-init](resources/user-data) is not sufficient,
+   you can generate new `cloud-init.iso` with the following steps:
+
+   1. Update content of [meta-data](resources/meta-data) file (the filename must be `meta-data`)
+   1. Update content of [user-data](resources/user-data) (the filename must be `user-data`)
+   1. Generate new iso with the cloud init by running:
+      ```sh
+      $ genisoimage -output cloud-init.iso -volid cidata -joliet -r user-data meta-data
+      ```
+   1. Upload the iso to the instance in case you already run the `init.sh` script:
+      ```
+      $ scp cloud-init.iso libvirtactuator:/var/lib/libvirt/images/cloud-init.iso
+      ```
+
+   *WARNING*: Please keep the `/sys/firmware/qemu_fw_cfg/by_name/opt/actuator.libvirt.io.k8s.sigs/config/raw` path
+   in the `user-data` file as it is. Otherwise, the user data (read bellow)
+   provided by kubernetes secret will get ignored.
+
+## How to prepare machine resources
+
+By default, available user data (under [user-data.sh](resources/user-data.sh) file)
+contains a shell script that deploys kubernetes master node.
+Feel free to modify the file to your needs.
+
+The libvirt actuator expectes the user data to be provided by a kubernetes secret
+by setting `spec.providerConfig.value.cloudInit.userDataSecret` field.
+See [userdata.yml](resources/userdata.yml) for example.
+
+At the same time, the `spec.providerConfig.value.uri` needs to be set to libvirt
+uri. E.g. `qemu+ssh://root@147.75.96.139/system` in case the libivrt instance
+is accessible via ssh on `147.75.96.139` IP address.
+
 ## To build the `libvirt-actuator` binary:
 
-You'll need to install libvirt-dev installed on the system you are building and running the binary. e.g. `apt-get -y install libvirt-dev`
+You'll need to install `libvirt-dev[el]` installed on the system you are building and running the binary.
+e.g. `apt-get -y install libvirt-dev` or `yum -y install libvirt-devel`
+
 ```sh
 CGO_ENABLED=1 go build -o bin/libvirt-actuator -a github.com/openshift/cluster-api-provider-libvirt/cmd/libvirt-actuator
 ```
 
-## Create libvirt instance based on machine manifest
+### Create libvirt instance based on machine manifest
 
 ```sh
 $ ./bin/libvirt-actuator create -m examples/machine.yaml -c examples/cluster.yaml
 ```
 
-Once the libvirt instance is created you can run `$ cat /tmp/test` to verify it contains the `Ahoj` string.
+Once the libvirt instance is created you can login inside (username: `fedora`, password: `fedora`):
+```
+$ virsh -c qemu+ssh://root@147.75.96.139/system console
+```
+
+Meantime you can check `/root/user-data.logs` to see the progress of deploying kubernetes master node:
+```
+$ watch -n 1 sudo tail -20 /root/user-data.logs
```

Comment:
Not sure I understand why you need to both watch and tail? Doesn't the latter effectively watch?